### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by f-hj**

### CODE REVIEW

File: lib/drive.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ function refreshMetrics(conf) {
       }
     }
 
-    total = Math.ceil((disk_info['1K-blocks'] * 1024)/ Math.pow(1024,2));
+    total = Math.ceil(((disk_info['1K-blocks'] | disk_info['1024-blocks']) * 1024)/ Math.pow(1024,2));
```
Comment (vmarchaud) [inline lib/drive.js]: `|` or `||` ?

File: lib/mem.js [hunk 2]
```diff
@@ -14,8 +15,16 @@ function refreshMetrics() {
     }
 
     var result_memory = (out.match(/\d+/g));
-    var total_mem = result_memory[0];
-    var free_mem = parseInt(result_memory[1]) + (parseInt(result_memory[3]) + parseInt(result_memory[4]));
+    var total_mem;
+    var free_mem;
+
+    if (os.platform == 'linux' || result_memory != null) {
```
Comment (vmarchaud) [inline lib/mem.js]: Can `freebsd` or `openbsd` works here too ?

File: lib/netstat.js [hunk 3]
```diff
@@ -70,27 +72,169 @@ function refreshNet(interval) {
...
-    value : function() { return 'N/A'; }
+    value : returnValue
   });
 }
 
+function returnValue() {
+  if (os.platform() == 'linux') {
+    return 'N/A';
+  } else {
+    return '❌';
```
Comment (vmarchaud) [inline lib/netstat.js]: why not put the cross in both case ?

File: lib/netstat.js [hunk 4]
```diff
@@ -70,27 +72,169 @@ function refreshNet(interval) {
...
+  }
+}
+
 function init(conf) {
   initMetrics();
 
-  refreshNet(conf.small_interval);
+  if (os.platform() == 'linux') {
```
Comment (vmarchaud) [inline lib/netstat.js]: same as above : `freebsd` & `openbsd` ?

File: lib/netstat.js (hunk 3 above)
Comment (f-hj) [inline lib/netstat.js]: ❌: OS incompatible, N/A: configuration incompatible

File: lib/drive.js (hunk 1 above)
Comment (f-hj) [inline lib/drive.js]: Corrected

File: lib/netstat.js (hunk 4 above)
Comment (f-hj) [inline lib/netstat.js]: Resolved for `freebsd` & `openbsd`

File: lib/netstat.js (hunk 3 above)
Comment (f-hj) [inline lib/netstat.js]: Resolved

File: lib/mem.js (hunk 2 above)
Comment (f-hj) [inline lib/mem.js]: Resolved

**Pull request merged**